### DISCUSSION TITLE
How to publish Minecraft mod?

### DISCUSSION THREAD

**Discussion by steamedhams42** [TEALSK12/teals-minecraft#24] [Q&A]
Students are having a great time with this project. We would like to know how to publish their mods for public consumption e.g. on Curseforge. I found this tutorial https://moddingtutorials.org/publishing/ but the `gradlew build` command does not seem to be recognized from this (`teals-minecraft`) repo.

**Comment by steamedhams42** [comment]
@hollasch Any solutions to this?

**Reply by hollasch** [reply]
Sorry, I'm not involved with this project these days.
